uli_init/simulate.py

```diff
@@ -481,20 +482,28 @@ def __init__(
         expand_factor=5
     ):
         self.molecule = molecule
-        self.para_weight = para_weight
         self.density = density
+        self.para_weight = para_weight
+        self.monomer_sequence = monomer_sequence
+        self.forcefield = forcefield
         self.target_L = None
-        self.remove_hydrogens = remove_hydrogens
         self.epsilon = epsilon
-        self.forcefield = forcefield
+        self.remove_hydrogens = remove_hydrogens
         self.assert_dihedrals = assert_dihedrals
         self.seed = seed
         self.system_mass = 0
         self.para = 0
         self.meta = 0
-        self.type = "melt"
         self.expand_factor = expand_factor
-
+        self.type = "melt"
+        
+        if self.monomer_sequence and self.para_weight:
+            raise ValueError(
```

Comment:
maybe add a test to check that this error is raised

Reply:
Done